common-service-core/src/main/java/org/zowe/apiml/security/HttpsFactory.java

```diff
@@ -54,18 +54,19 @@ public HttpsFactory(HttpsConfig httpsConfig) {
     }
 
 
-    public CloseableHttpClient createSecureHttpClient(HttpClientConnectionManager connectionManager) {
-
+    public CloseableHttpClient buildHttpClient(HttpClientConnectionManager connectionManager) {
         RequestConfig requestConfig = RequestConfig.custom()
-            .setConnectTimeout(config.getRequestConnectionTimeout())
-            .setSocketTimeout(config.getRequestConnectionTimeout())
-            .setConnectionRequestTimeout(config.getRequestConnectionTimeout()).build();
-        UserTokenHandler userTokenHandler = context -> context.getAttribute("my-token");
-
-        return HttpClientBuilder.create().setDefaultRequestConfig(requestConfig).setSSLHostnameVerifier(getHostnameVerifier())
-            .setConnectionTimeToLive(config.getTimeToLive(), TimeUnit.MILLISECONDS)
-            .setConnectionManager(connectionManager).disableCookieManagement().setUserTokenHandler(userTokenHandler)
+            .setConnectionRequestTimeout(Timeout.ofMilliseconds(config.getRequestConnectionTimeout()))
+        .build();
+       // UserTokenHandler userTokenHandler = context -> context.getAttribute("my-token");
+
+        return HttpClientBuilder.create().setDefaultRequestConfig(requestConfig)
+//           .setSSLHostnameVerifier(getHostnameVerifier())
+            .setConnectionManager(connectionManager).disableCookieManagement()
+//            .setUserTokenHandler(userTokenHandler)
```

Comment:
we have commented code here

Reply:
token handler returned, SSL is done in connection manager